changemakers/frappe_changemakers/doctype/payment_details/payment_details.json

```diff
@@ -0,0 +1,58 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2023-10-14 12:40:57.317152",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "sno",
+  "payment_type",
+  "payment_details",
+  "amount"
+ ],
+ "fields": [
+  {
+   "fieldname": "sno",
+   "fieldtype": "Int",
+   "in_list_view": 1,
+   "label": "Sno",
+   "reqd": 1
+  },
```

Comment:
Sno is not required in a child table, as it does the serialization of entries by itself